index.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+
+$language = "zh_CN";
+putenv("LANG=".$language);
+setlocale(LC_ALL, $language);
```

Comment:
On some systems, the locale is called `zh_CN.utf8`, so you should do something like this:
```php
setlocale(LC_ALL, [$language, $language.'.utf8']);
```